docker-compose.yml

```diff
@@ -4,32 +4,32 @@ services:
   fregepoc-redis:
     container_name: fregepoc-redis
     hostname: fregepoc-redis
-    image: redis:latest
+    image: redis:7.0
     networks:
       - fregepoc-main
     ports:
       - "16379:${DOCKER_REDIS_PORT}"
-    profiles: ["services", "application"]
+    profiles: [ "services", "application-dev", "application-prod" ]
     restart: on-failure
 
   fregepoc-postgres:
     container_name: fregepoc-postgres
     hostname: fregepoc-postgres
-    image: postgres:latest
+    image: postgres:14.2
     environment:
-       - POSTGRES_USER=${DJANGO_DATABASE_USER}
-       - POSTGRES_PASSWORD=${DJANGO_DATABASE_PASSWORD}
-       - POSTGRES_DB=${DJANGO_DATABASE_NAME}
+      - POSTGRES_USER=${DJANGO_DATABASE_USER}
+      - POSTGRES_PASSWORD=${DJANGO_DATABASE_PASSWORD}
+      - POSTGRES_DB=${DJANGO_DATABASE_NAME}
     networks:
       - fregepoc-main
     ports:
       - "15432:${DOCKER_POSTGRES_PORT}"
-    profiles: ["services", "application"]
+    profiles: [ "services", "application-dev", "application-prod" ]
```

Comment:
We can simplify `application-dev` and `application-prod` to `dev` and `prod`, respectively. There is no need for the `application` prefix other than legacy.